### PR TITLE
analysis docs

### LINKED ISSUE #628: create detailed documentation for haddock3-analyse
haddock3-analyse should now have an adequate, detailed documentation

### PR DESCRIPTION
**Pull request by mgiulini**
You are about to submit a new Pull Request. Before continuing make sure you read the [contributing guidelines](CONTRIBUTING.md) and that you comply with the following criteria:

- [ ] You have sticked to Python. Please talk to us before adding other programming languages to HADDOCK3
- [ ] Your PR is about CNS
- [ ] Your code is well documented: proper docstrings and explanatory comments for those tricky parts
- [ ] You structured the code into small functions as much as possible. You can use classes if there is a (state) purpose
- [ ] Your code follows our coding style
- [ ] You wrote tests for the new code
- [ ] `tox` tests pass. *Run `tox` command inside the repository folder*
- [ ] `-test.cfg` examples execute without errors. *Inside `examples/` run `python run_tests.py -b`*
- [ ] PR does not add any dependencies, unless permission granted by the HADDOCK team
- [ ] PR does not break licensing
- [ ] Your PR is about writing documentation for already existing code :fire:
- [ ] Your PR is about writing tests for already existing code :godmode:

---

Closes #628 by adding a new .md file in the `docs/tutorials` folder, namely `analysing_runs.md`. It describes the basic usage of `haddock3-analyse` and `haddock3-traceback`. I also added a couple of example plots for `haddock3-analyse`

### CODE REVIEW

File: docs/tutorials/analysing_runs.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+a `caprieval` module, `haddock3-analyse` simply loads the `capri_ss.tsv` and `capri_clt.tsv` files
+produced by the `caprieval` module. Otherwise, `haddock3-analyse` runs a `caprieval` analysis of the models.
+You can provide some [caprieval-specific parameters](https://github.com/haddocking/haddock3/blob/main/src/haddock/modules/analysis/caprieval/defaults.yaml)
+using the following syntax:
+
+```
+haddock3-analyse -r my-run-folder -m 2 5 6 -p reference_fname my_ref.pdb receptor_chain F
+```
+
+Here the `-p` key tells the code that you are about to insert caprieval parameters, whose name should match the parameter name.
```
Comment (VGPReys) [inline docs/tutorials/analysing_runs.md:25]: parameter name and corresponding value must be separated by space.

File: docs/tutorials/analysing_runs.md [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+
+```
+ ====================================================================
+# This is a HADDOCK3 configuration file
+
+# directory in which the docking will be done
+run_dir = "my-run-folder"
+
+# postprocess the run
+postprocess = true
```
Comment (VGPReys) [inline docs/tutorials/analysing_runs.md:132]: Open question (that should be discussed):
shoudn't the value of the `postprocess` parameter be true by default ?

File: docs/tutorials/analysing_runs.md
Comment (VGPReys) [inline docs/tutorials/analysing_runs.md:1]: What about placing the `post-processing` section in first ? and then explaining the `haddock3-analyse` and `haddock3-traceback` ? 

Also adding internal links to the 3 sections that will be later presented ?

```
Dedicated analyses modules are made available, allowing to investigate the different steps of a HADDOCK3 workflow, even after it has been completed.
* [Post-processing](#the-postprocess-option)  
* [Analyse](#haddock3-analyse)
* [Traceback](#haddock3-traceback)
``` ?

Comment (mgiulini) [reply docs/tutorials/analysing_runs.md:1]: actually I would not modify the titles, I think that it's beneficial to have "haddock3-analyse" as a title for the user

**Review by rvhonorato — COMMENTED** [review]
What you are proposing is **user** documentation, but the docs that are built automatically are related to **code** documentation. 

I would not mix these two in the same codebase because user access can be limited by documentation build issues that are unrelated to the information provided in those docs.

User information should fit together with the rest of the material at bonvinlab.org. However this is not for me to decide since I'm not the lead on the project.

**Comment by mgiulini** [comment]
> What you are proposing is **user** documentation, but the docs that are built automatically are related to **code** documentation.
> 
> I would not mix these two in the same codebase because user access can be limited by documentation build issues that are unrelated to the information provided in those docs.
> 
> User information should fit together with the rest of the material at bonvinlab.org. However this is not for me to decide since I'm not the lead on the project.

I know this is user documentation, of course. It was thought as a technical guide for people that want to approach the haddock3-specific analysis, as it's done in the other docs/tutorials/ markdown files.
I understand your point about decoupling the repo and the user docs, though I find that having a few technical tutorials within the repo is still useful, especially for developers..I guess we should talk about this with @amjjbonvin at some point

**Comment by mgiulini** [comment]
moved to https://github.com/haddocking/haddocking.github.io/pull/677

**Pull request closed without merging**